test/turbopack-tests-manifest.json

```diff
@@ -2312,10 +2312,10 @@
     "runtimeError": false
   },
   "test/e2e/app-dir/app-compilation/index.test.ts": {
-    "passed": ["app dir HMR should not cause error when removing loading.js"],
+    "passed": [],
     "failed": [],
     "pending": [],
-    "flakey": [],
+    "flakey": ["app dir HMR should not cause error when removing loading.js"],
```

Comment:
also add it to this list, otherwise a manifest update would enable the test again: https://github.com/vercel/next.js/blob/81f9e3e5d29eb76f229df76b223d1426841c3cb8/test/build-turbopack-tests-manifest.js#L25